src/java/com/unifina/feed/AbstractFeedProxy.java

```diff
@@ -30,18 +28,10 @@
 		extends AbstractFeed<ModuleClass, MessageClass, KeyClass, EventRecipientClass>
 		implements MessageRecipient<RawMessageClass, KeyClass>, ICatchupFeed {
 
-	enum CatchupState {
-		CATCHUP,
-		CATCHUP_UNSYNC_READY
-	}
-
 	private static final Logger log = Logger.getLogger(AbstractFeedProxy.class);
 
 	private final MessageHub<RawMessageClass, MessageClass, KeyClass> hub;
-	private final Queue<MessageClass> realtimeWaitQueue = new ConcurrentLinkedQueue<>();
-	private final Queue<Long> realtimeWaitQueueCounter = new ConcurrentLinkedQueue<>();
 
-	private CatchupState catchupState = CatchupState.CATCHUP; // start in catchup state
 	private Catchup catchup = null;
```

Comment:
This variable is related to the Catchup feature and could be removed. (Maybe the whole `Catchup` class can be deleted`?)